hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/security/x509/certificate/authority/CertificateStore.java

```diff
@@ -106,6 +106,13 @@ Optional<Long> revokeCertificates(List<BigInteger> serialIDs,
    */
   void removeExpiredCertificate(BigInteger serialID) throws IOException;
 
+  /**
+   * Deletes all non-revoked expired certificates from the store.
+   *
+   * @throws IOException
+   */
+  void removeAllExpiredCertificates();
```

Comment:
This need go through ratis. All APIs go through ratis requires annotation "@Replicate".